code/game/gamemodes/cult/runes.dm

```diff
@@ -851,14 +867,14 @@ var/list/sacrificed = list()
 		if(istype(cultist.loc, /obj/machinery/dna_scannernew)&&cultist.loc:locked)
 			cultist.loc:locked = 0
 		for(var/mob/living/carbon/C in users)
-			user.take_overall_damage(dam, 0)
-			C.say("Khari[pick("'","`")]d! Gual'te nikka!")
+			user.rune_apply_damage(10, 15)

```

Comment:
Non-humans frankly shouldn't be using runes in the first place.  I plan to eventually any nonhuman mob out of cult entirely (why would a diona nymph or mouse or something be a cultist?), and there is no point in coding in exceptions for something almost never (possibly never period) happens in something I'm gradually working on.

It's not a bug, and it's not going to runtime.

Reply:
It's still dumb design. Understand. This will deal/remove the amount of blood equivalent to n * x, where n is the amount of people using the rune. _This means that it's a static amount._ There's no positive to be gained from having more than 3 people (the minimum required), and yet you're applying a penalty for it by removing more blood if there's more people.

The old code would consistently apply 15 units of damage, no matter how many people were using the rune. You need to have that behaviour for this rune. So again: either restore the old behaviour, or move this statement to be before the for loop.